### DISCUSSION TITLE
Only first one item from whole stack has been marked for deconstruct or ignoring by bots.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13837] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

1. When there are a stack of items on the floor, and you give an order to deconstruct this stack,
then only the first of item is marked for deconstruction.
2. From stack of items in the inventory also only one was marked for deconstruction,.
3. From stack of items in the cabinet  also only one was marked for deconstruction.
I had tried this on a stack of minerals.

### Reproduction steps

Drop stack of items on the floor or 
put it to your inventory, mark "Deconstruct" and then drop
Give "Deconstruct order"
You will see bot taken only one item of whole stack and brought it to deconstruct.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I [created a ticket about this](https://github.com/FakeFishGames/Barotrauma/issues/13843), we'll fix it in a future patch.

**Comment by Rokvach** [comment]
The issue has been fixed and tested to be working, closing discussion.

**Reply by pavel4win** [reply]
Not has been fixed yet in network game. Look at https://github.com/FakeFishGames/Barotrauma/discussions/14089

**Reply by Regalis11** [reply]
~~The fix is not yet included in the public version of the game.~~

EDIT: no, it seems it was included in the hotfix! So it does seem like there's an issue here still, sorry, I'll reopen the other ticket.
